charts/firefly/templates/core/prometheusrule.yaml

```diff
@@ -0,0 +1,80 @@
+{{/*
+  Copyright © 2022 Kaleido, Inc.
+
+  SPDX-License-Identifier: Apache-2.0
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://swww.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/}}
+
+{{- if and .Values.core.metrics.alertRule.enabled .Values.config.metricsEnabled }}
+{{- if not (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") }}
+# WARNING: prometheus-operator is not installed but alertRule has been enabled, this will fail. Please install
+#          prometheus-operator to resolve this.
+{{- end }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "firefly.fullname" . }}
+  labels:
+  {{- include "firefly.coreLabels" . | nindent 4 }}
+spec:
+  groups:
+    - name: {{ include "firefly.fullname" . }}
+      rules:
+        - alert: {{ include "firefly.fullname" . }}TargetDown
```

Comment:
Also since fullnames are often lower case i.e. `firefly` this will be something like `fireflyTargetDown`...

Not sure if templating it is really necessary or not. I think the group `name` will handle making it unique? But might take some testing. No need to spend a ton of time on this but just wanted to point out most alert names are capitalized looking at the default promrules that come with the operator.